pkgs/dart_tooling_mcp_server/test/test_harness.dart

```diff
@@ -119,6 +114,81 @@ class TestHarness {
   }
 }
 
+/// The debug session for a single app - has its own dtd instance and
```

Comment:
do we want this to continue having its own DTD instance? If we want to support a multi-app use case, then we'd want 1 dtd instance to 1 or many vm service connections

Reply:
Sorry this comment is out of date, will remove :). I think we want always one primary DTD instance